webapp-django/crashstats/documentation/jinja2/documentation/supersearch/home.html

```diff
@@ -52,7 +52,7 @@ <h2 id="supersearchunredacted">SuperSearchUnredacted</h2>
     <h1 id="basic-usage">Basic Usage</h1>
 
     <div class="example">
-      <pre><code data-url="{{ url('api:model_wrapper', model_name='SuperSearch') }}?{{ make_query_string(product=product, _results_number=10) }}"><span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?{{ make_query_string(product=product, _results_number=10) }}</span></code></pre>
+      <pre><code data-url="{{ url('api:model_wrapper', model_name='SuperSearch') }}?{{ make_query_string(product=product.name, _results_number=10) }}"><span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?{{ make_query_string(product=product.name, _results_number=10) }}</span></code></pre>
```

Comment:
It's interesting to note that this page no longer works if you don't have enough crash report data to cover the default product. I'm not sure what to do about that. While I think this page is somewhat fancy, I think I'd rather it contained hard-coded examples.